automation/tdx_canonical_verifier.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+CUR_DIR=$(pwd)
+TDX_CONFIG=tdx-config
+TDX_DIR="$CUR_DIR"/tdx_verifier
+source "$CUR_DIR"/tdx_canonical_common
+source "$CUR_DIR"/$TDX_CONFIG
+GUEST_TOOLS_DIR="$TDX_DIR"/guest-tools
+GUEST_IMG_DIR="$GUEST_TOOLS_DIR"/image
+TD_GUEST_VERIFY_TEXT="tdx: Guest detected"

```

Comment:
Do we need variables for the messages? Can't we just compare it directly with the string at the place? In general all variables from line 1-17 if they are only used once then no need to set any variable but directly use the string.

Reply:
Removed below variables as this was used only once in the script.
TD_GUEST_VERIFY_TEXT
TDT_HOST_VERIFY_TEXT
SERVICE_ACTIVE_STATUS_VERIFY_TEXT

Retained MPA_SERVICE_CHECK and MPA_REGISTRATION_CHECK as comparison string is bigger.
rest all other variables are used more than once.

moved pccs keys check functionlity to configure_attestation_host
moved Intel Tiber Trust Service API key check functionlity to attest_intel_tiber_trust_service